package szarp;

message ParamValue {
	required uint32 param_no = 1;
	required uint32 timestamp = 2;
	optional uint32 utimestamp = 3;

	optional uint32 int_value = 11;
	optional float float_value = 12;
	optional double double_value = 13;
}

message ParamsValues {
	repeated ParamValue paramValues = 1;
}
